modules/020-deckhouse/docs/USAGE.md

```diff
@@ -76,6 +76,24 @@ Manual confirmation of the update to the version `v1.26.0`:
 kubectl patch DeckhouseRelease v1-26-0 --type=merge -p='{"approved": true}'
 ```
 
+### Manual disruption update confirmation
+
+If necessary, it is possible to enable manual confirmation of disruptive updates (updates that change the default values or behavior). This can be done as follows:
+
+```yaml
+deckhouse: |
+  ...
+  releaseChannel: Stable
+  update:
+    disruptionApproval: Manual
+```
+
+In this mode, it will be necessary to confirm each minor disruptive update with an annotation:
+
+```shell
+kubectl annotate DeckhouseRelease v1-36-0 release.deckhouse.io/disruption-approved=true
```

Comment:
How does it work if I have both `update.disruptionApproval` and `update.approvalMode` set to `Manual`? Do I need to approve release twice? Maybe there is something we can optimize in this case.

Reply:
Yes, we will approve twice. It's different flows we don't need to mix them.
First: we approve a release
Second: if the release has disruption changes - we approve our confirmation (after one more brain storm process ;)